tests/helper.py

```diff
@@ -2,6 +2,195 @@
 Helper functions for unit tests.
 """
 
+import unittest
+
+import geomstats.backend as gs
+
+
+class TestGeomstatsMethods(unittest.TestCase):
+    def assertScalar(self, result, n_samples=1):
+        return self.assertTrue(
+            gs.allclose(result.shape, (n_samples, 1)),
+            '\nresult.shape = {} instead of {}'.format(
+                result.shape, (n_samples, 1)))
+
+    def assertVector(self, result, n_samples=1, dim=1):
+        return self.assertTrue(
+            gs.allclose(result.shape, (n_samples, dim)),
+            '\nresult.shape = {} instead of {}'.format(
+                result.shape, (n_samples, dim)))
+
+    def assertAllClose(self, result, expected):
```

Comment:
we should actually be using a backend function for this - that works like 

np.testing.assert_almost_equal(x, y)

Reply:
I'm using the backend function `gs.allclose` but I wanted to encompass it as a unittest's function.

Should we use the other backend function that you suggest? Or not encompassing it as a unittest' function?